graylog2-server/src/test/java/org/graylog2/configuration/HttpConfigurationTest.java

```diff
@@ -61,6 +62,7 @@ public void testHttpBindAddressIsIPv6Address() throws RepositoryException, Valid
     }
 
     @Test
+    @Ignore // Disabled test due to being unreliable (see https://github.com/Graylog2/graylog2-server/issues/4459)
```

Comment:
The [`@Ignore`](http://junit.sourceforge.net/javadoc/org/junit/Ignore.html) annotation has an argument which can be used to display the reason while running the test.
This might be more useful than a code comment.

Reply:
Indeed!